wire-ios-sync-engine/Tests/Source/SessionManager/SessionManagerTests.swift

```diff
@@ -396,6 +396,27 @@ final class SessionManagerTests: IntegrationTest {
         }
     }
 
+    func tmpDirectoryPath() -> URL {
+        return URL(fileURLWithPath: NSTemporaryDirectory())
+    }
+
+    func testThatItDestroyedTmpDirectoryAfterLoggedOut() {
+
+        // GIVEN
+        XCTAssertTrue(login())
+        let observer = SessionManagerObserverMock()
+        let token = sessionManager?.addSessionManagerDestroyedSessionObserver(observer)
```

Comment:
I suggest to add something to the temp directory and assert it's not empty, then we can be sure that after logging out the directory was emptied.

Reply:
yes, added test file to double check deletion